src/tests/EdFi.DataManagementService.Tests.E2E/Hooks/SetupHooks.cs

```diff
@@ -50,7 +52,7 @@ public static async Task BeforeFeature(PlaywrightContext context, TestLogger log
                 logger.log.Debug("Using local environment, verify that it's correctly set.");
             }
 
-            await context.CreateApiContext();
+            await context.InitializeApiContext();
```

Comment:
This is called many times, replacing an existing context every time except for the first execution. Therefore "initialize" seemed like a better verb than "create".